village/src/components/Smurf.js

```diff
@@ -1,13 +1,36 @@
 import React from 'react';
 
-const Smurf = props => {
+
+// passed as props from Smurfs.js
+// name={smurf.name}
+// id={smurf.id}
+// age={smurf.age}
+// height={smurf.height}
+// key={smurf.id}
+//deleteSmurf={props.deleteSmurf}
+
+
+
+class Smurf extends React.Component {
+
+  delete = (id) => {
+    this.props.deleteSmurf(id)
+  }
```

Comment:
Out of curiosity, why did you choose to call this method inside of another method, instead of just calling the method directly in the event listener?